src/web/tools/GestureHandlerOrchestrator.ts

```diff
@@ -287,10 +287,7 @@ export default class GestureHandlerOrchestrator {
       return false;
     }
 
-    if (
-      handler !== otherHandler &&
-      (handler.isAwaiting() || handler.getState() === State.ACTIVE)
-    ) {
+    if (handler.isAwaiting() || handler.getState() === State.ACTIVE) {
```

Comment:
This one seems a bit weird. Why do we want a handler to be cancellable by itself?

Reply:
It won't be cancelled by itself. The thing is, I've written it exactly how it's done in Android implementation, but if you look at [this line](https://github.com/software-mansion/react-native-gesture-handler/blob/a0506c026fd4a73238b491a1bb0a6b6c2b973ad3/src/web/tools/GestureHandlerOrchestrator.ts#L276C11-L276C11) you'll notice that this check is not necessary.